index.html

```diff
@@ -44,7 +204,49 @@ <h1> I’m George
                 <li> <a href="#"><img src="img/Vectorch.png" alt="icon"></a> </li>
             </ul>
         </div>
+        <a href="#"> <div class="btn-main" id="reume"> Get my resume</div> </a> 
+        <br>
+        
+        <div id="skills">
+            <ul id="skill">
+                <li>
+                    <div class="language">
+                        <P><b>Languages</b></P>
+                        <p id="rotate"> > </p>
+                    </div>
+
+                    <br><br>
+            <div class="lang-list">
+                <ul class="a-items">
+                    <li class="b-items"> <img src="img/js.png"> <a href="#" id="icon-text"> JavaScrip</a> </li>
+                    <li class="b-items"> <img src="img/html.png"> <a href="#" id="icon-text"> HTML</a> </li>
+                    <li class="b-items"> <img src="img/css.png"> <a href="#" id="icon-text"> CSS</a> </li>                        
+                </ul>                
+            </div>                    
+                </li>
+                <li>
+                    <div class="frame-8">
+                        <ul class="frameworks">
+                            <li><p class="frames">Frameworks</p></li>
+                            <li><img src="img/Disabledforward.png" alt="forward" class="forward"></li>
+                        </ul>
+                                   
+                        <div class="line"></div>
+                    </div>
+                </li>
+                <li>
+                    <div class="frame-8">
+                        <ul class="frameworks">
+                            <li><p class="frames">Skills</p></li>
+                            <li><img src="img/Disabledforward.png" alt="forward" class="forward"></li>
+                        </ul>
+                                   
+                        <div class="line"></div>
```

Comment:
- [ ] Kindly align these two correctly so that your design might match the chosen Figma design

**This is your design right now**

![image](https://user-images.githubusercontent.com/86318284/193090318-cd121ae3-85cb-4518-908e-5391654b5412.png)

**This is the desired design**

![image](https://user-images.githubusercontent.com/86318284/193090402-25d522ac-01b9-4363-8355-67aa42dd554f.png)